<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>robotest</class>
 <widget class="QMainWindow" name="robotest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>robotest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Исх. углы сервоприводов, °</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>M1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>M2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>M3</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="e3Spin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="e2Spin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="e1Spin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Углы вращ. базовой точки, °</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>OX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>OY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>OZ</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="zSpin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ySpin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="xSpin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Корр. углы сервоприводов, °</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>M1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>M2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>M3</string>
     </property>
    </widget>
    <widget class="QLabel" name="m3Gr">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>М3 поворот</string>
     </property>
    </widget>
    <widget class="QLabel" name="m2Gr">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>М2 поворот</string>
     </property>
    </widget>
    <widget class="QLabel" name="m1Gr">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>М1 поворот</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Новые координаты сегмента l3</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3y1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3y1</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3y2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3y2</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3z1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3z1</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3x1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3x1</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3z2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3z2</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3x2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3x2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Исходные координаты сегмента l3</string>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3y2_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3y2</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3y1_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3y1</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3z2_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3z2</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3z1_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3z1</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3x1_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3x1</string>
     </property>
    </widget>
    <widget class="QLabel" name="v3x2_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>v3x2</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
